samples/ComponentExamples/package.json

```diff
@@ -68,4 +68,4 @@
     "webpack-cli": "~4.10.0",
     "webpack-dev-server": "4.8.1"
   }
-}
\ No newline at end of file
+}
```

Comment:
And some missing line switch at the end of json file (I guess there is a eslint difference)